### PR TITLE
v2: Run the reference benchmark on Travis

### PR DESCRIPTION
**Pull request by rtfb**
So that we could have a historic record of performance.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,3 +16,4 @@ install:
 
 script:
     - go test -v ./...
+    - go test -short -bench BenchmarkReference -benchmem
```
Comment (shurcooL) [inline .travis.yml]: What is `-short` flag for?

You probably want this line to only run that `BenchmarkReference` benchmark and no tests, right?

If so, I think you should add `-run=^$` or something equivalent.

Comment (rtfb) [inline .travis.yml]: You're right, fixed.

File: .travis.yml [hunk 2]
```diff
@@ -16,3 +16,4 @@ install:
 
 script:
     - go test -v ./...
+    - go test -run=^$ -bench BenchmarkReference -benchmem
```
Comment (shurcooL) [inline .travis.yml]: Really minor and optional, but I'd consider doing including `=` in `-bench=BenchmarkReference` for consistency with `-run=^$`. Up to you.

**Comment by shurcooL** [comment]
Minor optional comment, LGTM.

**Pull request merged**